### DISCUSSION TITLE
Fastlane finished with errors: Exit status: 165

### DISCUSSION THREAD

**Discussion by panncherry** [fastlane/fastlane#17065] [Q&A]
Summary for scan 2.156.1   

Executed 10 tests, with 0 failures (0 unexpected) in 0.014 (0.024) seconds
Executed 5 tests, with 0 failures (0 unexpected) in 139.983 (139.989) seconds

Test Succeeded  but Fastlane finished with errors: Exit status: 165

<img width="1226" alt="Screen Shot 2020-08-17 at 11 09 55 PM" src="https://user-images.githubusercontent.com/17572436/90466051-c58dfd80-e0de-11ea-8d8d-52e21d3eacc7.png">

<img width="1128" alt="Screen Shot 2020-08-17 at 11 09 35 PM" src="https://user-images.githubusercontent.com/17572436/90466035-b9a23b80-e0de-11ea-9fd6-1d5541b3fe11.png">

FastFile
```
# fastlane/Fastfile
default_platform(:ios)

platform :ios do
  before_all do
    setup_circle_ci
  end

  desc "Run all the tests"
  lane :test do
    scan
  end
end
```

**Comment by Thompsons21** [comment]
I have managed to solve this problem by creating a fastlane folder in the root folder of my react-native project and inside that I have initiated the fastlane command. Before I used to have the fastlane folder inside iOS folder. [My Penn Medicine](https://www.mypennmedicine.ltd/)
